virtio-queue/Cargo.toml

```diff
@@ -23,6 +23,19 @@ criterion = "0.6.0"
 vm-memory = { workspace = true, features = ["backend-mmap", "backend-atomic"] }
 memoffset = "0.9.0"
 
+# The following deps are needed only under kani
```

Comment:
nit: we can remove this comment, it isn't adding any information more than the following code.

Just a suggestion for the future, use comments more to explain the reason, the "why" of code/configuration or to explain unclear part. In this case the section is very clear and the comment will not add more.

Reply:
acknowledged